nix/s3-bucket.nix

```
@@ -40,6 +40,12 @@ with lib;
       description = "The JSON lifecycle management string to apply to the bucket.";
     };
 
+    versioning = mkOption {
+      default = "Suspended";
+      type = types.str;
```

Instead of a string, should use an enum: https://github.com/NixOS/nixops/blob/28231a177d751e800af3223a8763ea75b0ef9dd9/nix/cloudwatch-metric-alarm.nix#L86

---

```suggestion
      type = types.enum [ "Suspended" "Enabled" ];
```

should do it, I think (please test it).

---

Thanks, Updated ( and tested)